src/Resources/translations/SonataAdminBundle.fr.xliff

```diff
@@ -446,6 +446,10 @@
         <source>link_actions</source>
         <target>Liste d'actions</target>
       </trans-unit>
+      <trans-unit id="list.label__actions">
+        <source>list.label__actions</source>
+        <target>Liste d'actions</target>
```

Comment:
Liste d'actions would be translated as `Actions List` in english ; but we use `Actions` in english.
So I would use just
```suggestion
        <target>Actions</target>
```
here.

And I wonder if we shouldn't do the same for link_actions.

Reply:
Not sure, feel free to apply the modifications you consider, I just copied the link_actions.